src/external/SMIOL/smiol.c

```diff
@@ -336,10 +336,12 @@ int SMIOL_open_file(struct SMIOL_context *context, const char *filename,
 				MPI_Comm_free(&io_group_comm);
 				return SMIOL_INVALID_FORMAT;
 			}
-
+            MPI_Info info;
+            MPI_Info_create(&info);
+            MPI_Info_set(info, "nc_header_align_size", "131072");
```

Comment:
So that it's easier to find and modify in future, I'd suggest declaring the string `"131072"` as a macro at the top of the file where we also define `MAX_REQS`.